_source/logzio_collections/_metrics-sources/Prometheus-cloudwatch-exporter.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Ship Prometheus metrics from AWS CloudWatch with the OpenTelemetry Collector
+logo:
+  logofile: prometheusio-icon.svg
+  orientation: vertical
+data-source: Prometheus
+templates: ["docker-metricbeat"]
+open-source:
+  - title: 
+    github-repo: 
+contributors:
+  - yotamloe
+  - yberlinger
+shipping-tags:
+  - prometheus
+---
+
+# logzio-aws-metrics
+
+**IMPORTANT ⚠️ :** This project is intended for specific private beta customers.
+
+This project allows you to collect Prometheus format metrics from Amazon CloudWatch with the CloudWatch exporter, and ship them to Logz.io using the OpenTelemetry collector.
+
+### Schema
+![Prometheus-CloudFront-OpenTelemetry schema](https://dytvr9ot2sszz.cloudfront.net/logz-docs/metrics-prometheus/aws-docker-v2.png)
```

Comment:
We need to remove the black line surrounding the image.
![chrome-capture - 2021-01-17T192613 396](https://user-images.githubusercontent.com/48534558/104850808-88c66c80-58f1-11eb-85b1-2f114cef79bb.jpg)

Reply:
I've asked one of the UX designers to have a look.